src/components/Pop/styles/PopArrow.css.js

```diff
@@ -33,47 +33,47 @@ const css = (props: Object) => {
   `
 
   return `
-    ${baseStyles}
-    ${borderColor}
-    border-style: solid;
-    height: 0;
+    ${baseStyles};
+    ${borderColor};
+    height: ${sizePx};
     position: absolute;
-    width: 0;
+    transform: rotate(45deg);
+    width: ${sizePx};
 
     &.is-top {
-      border-width: ${sizePx} ${sizePx} 0 ${sizePx};
-      border-left-color: transparent;
-      border-right-color: transparent;
-      border-bottom-color: transparent;
-      bottom: -${sizePx};
-      ${verticalStartEnd}
+      bottom: calc((${sizePx} / 2) * -1);
+      ${verticalStartEnd};
+
+      &.is-ghost {
+        margin-bottom: 1px;
+      }
     }
 
     &.is-bottom {
-      border-width: 0 ${sizePx} ${sizePx} ${sizePx};
```

Comment:
No more border tricks to render the arrow! It's now a square rotated 45deg. Massively simpler and more reliable